router/sources/router.move

```diff
@@ -527,19 +527,23 @@ module router::router {
         }
     }
 
-    public entry fun set_subdomain_expiration_policy(
-        _domain_admin: &signer,
-        _domain_name: String,
-        _subdomain_name: String,
-        _expiration_policy: u8,
+    public entry fun domain_admin_set_subdomain_expiration_policy(
+        domain_admin: &signer,
+        domain_name: String,
+        subdomain_name: String,
+        expiration_policy: u8,
     ) acquires RouterConfig {
         let mode = get_mode();
         if (mode == MODE_V1) {
             // Will not be implemented in v1
             abort error::not_implemented(ENOT_IMPLEMENTED_IN_MODE)
         } else if (mode == MODE_V1_AND_V2) {
```

Comment:
do you think we should have a `mode == MODE_V2` in place now so that we don't have to revise when V1 retires